docker/dev/docker.env

```diff
@@ -7,8 +7,11 @@ DEBUG=True
 DJANGO_SETTINGS_MODULE=settings.dev
 DJANGO_SERVER=django
 
+EVALAI_HOST_URL=http://127.0.0.1:8888
```

Comment:
Please change the name to `HOSTNAME` and the value should be `localhost:8888`/`evalai.cloudcv.org`/`evalai-staging.cloudcv.org`